From a5f30dbf50c6478c53b4fb711c0490346e34cdea Mon Sep 17 00:00:00 2001
From: Mike Thompson <mthompson@dustyrobotics.com>
Date: Mon, 18 Apr 2022 23:34:43 -0700
Subject: [PATCH 1/1] Fixed linking error with yyloc

---
 scripts/dtc/dtc-parser.tab.c_shipped | 2 --
 1 file changed, 2 deletions(-)

diff --git a/scripts/dtc/dtc-parser.tab.c_shipped b/scripts/dtc/dtc-parser.tab.c_shipped
index aea514fa6928..3da9433341ba 100644
--- a/scripts/dtc/dtc-parser.tab.c_shipped
+++ b/scripts/dtc/dtc-parser.tab.c_shipped
@@ -1201,8 +1201,6 @@ int yychar;
 
 /* The semantic value of the lookahead symbol.  */
 YYSTYPE yylval;
-/* Location data for the lookahead symbol.  */
-YYLTYPE yylloc
 # if defined YYLTYPE_IS_TRIVIAL && YYLTYPE_IS_TRIVIAL
   = { 1, 1, 1, 1 }
 # endif
-- 
2.25.1

