From d1c0a474a8987fb85de4e7b21713a757491a31eb Mon Sep 17 00:00:00 2001
From: Mike Thompson <mthompson@dustyrobotics.com>
Date: Tue, 7 Jun 2022 10:16:22 -0700
Subject: [PATCH 1/1] Removed troublesome .dtb files

---
 arch/arm/dts/Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 594614a2c6..84c7312644 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -28,11 +28,9 @@ dtb-$(CONFIG_EXYNOS5) += exynos5250-arndale.dtb \
 	exynos5422-odroidxu3.dtb
 dtb-$(CONFIG_EXYNOS7420) += exynos7420-espresso7420.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += \
-	rk3036-sdk.dtb \
 	rk3066a-mk808.dtb \
 	rk3126-evb.dtb \
 	rk3128-evb.dtb \
-	rk3188-radxarock.dtb \
 	rk3288-evb.dtb \
 	rk3288-fennec.dtb \
 	rk3288-firefly.dtb \
@@ -47,16 +45,18 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += \
 	rk3288-vyasa.dtb \
 	rk3308-evb.dtb \
-	rk3328-evb.dtb \
-	rk3368-lion.dtb \
-	rk3368-sheep.dtb \
-	rk3368-geekbox.dtb \
-	rk3368-px5-evb.dtb \
 	rk3399-evb.dtb \
 	rk3399-firefly.dtb \
 	rk3399-puma-ddr1333.dtb \
 	rk3399-puma-ddr1600.dtb \
 	rk3399-puma-ddr1866.dtb \
 	rv1108-evb.dtb
+#	rk3036-sdk.dtb \
+#	rk3188-radxarock.dtb \
+#	rk3328-evb.dtb \
+#	rk3368-lion.dtb \
+#	rk3368-sheep.dtb \
+#	rk3368-geekbox.dtb \
+#	rk3368-px5-evb.dtb \
 dtb-$(CONFIG_ARCH_MESON) += \
 	meson-gxbb-odroidc2.dtb
 dtb-$(CONFIG_TEGRA) += tegra20-harmony.dtb \
-- 
2.34.1

