From 6470623a248cc90bea1594656737e45c71247588 Mon Sep 17 00:00:00 2001
From: Mike Thompson <mthompson@dustyrobotics.com>
Date: Wed, 19 Apr 2023 02:30:29 +0000
Subject: [PATCH 1/1] Silenced address of packed member warning

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index de68fa12be..bed0bc6dc0 100644
--- a/Makefile
+++ b/Makefile
@@ -359,6 +359,7 @@ KBUILD_CPPFLAGS := -D__KERNEL__ -D__UBOOT__
 
 KBUILD_CFLAGS   := -Wall -Wstrict-prototypes \
 		   -Wno-format-security \
+		   -Wno-address-of-packed-member \
 		   -fno-builtin -ffreestanding
 KBUILD_CFLAGS	+= -fshort-wchar -Werror
 KBUILD_AFLAGS   := -D__ASSEMBLY__
-- 
2.34.1

